docs/editor_manual/copying_aliasing_existing_pages.md

```diff
@@ -18,8 +18,6 @@ Here is how to copy an existing page:
 
 -   Once you completed this form, press _Copy this page_. The page will then be copied and appear in the Explorer page.
 
-![Successful copying of a page results in it being available in the Explorer page.](../_static/images/screen12.6_3_copy_success.png)
```

Comment:
Removed because this was very painful to set up (one-off screenshot taking of a success message on a listing requires deleting any precedent attempts beforehand). This didn’t feel useful enough of a visual to warrant the effort.